templates/api-index.js

```diff
@@ -49,6 +49,36 @@ const handleError = (err) => {
   };
 }
 
+const scrub = (data) => {
+  if (!data || (typeof data !== 'object' && !Array.isArray(data))) return data;
```

Comment:
```suggestion
  if (!data || typeof data !== 'object') return data;
```
This is all you need. `typeof [] === 'object'` Isn't JavaScript fun??? 😛